code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -35,41 +35,139 @@
 	id = "nicotine"
 	description = "Slightly reduces stun times. If overdosed it will deal toxin and oxygen damage."
 	reagent_state = LIQUID
-	color = "#60A584" // rgb: 96, 165, 132
-	addiction_threshold = 30
+	color = rgb(0,0,0)
+	overdose_threshold = 30
 
 /datum/reagent/drug/nicotine/on_mob_life(mob/living/M)
 	if(prob(1))
 		var/smoke_message = pick("You feel relaxed.", "You feel calmed.","You feel alert.","You feel rugged.")
 		M << "<span class='notice'>[smoke_message]</span>"
-	M.AdjustStunned(-1, 0)
+	if(prob(50))
+		M.Jitter(5)
```

Comment:
Keep in mind, Goon's Jitter is different than here--there, your jitter has to build up to a specific number before you'll start twitching...here, you start twitching instantly; may not want this to kick in until it's been in your bloodstream for so many cycles.